adminkuizlist.php

```diff
@@ -47,7 +47,7 @@
     });
   });
  
-    $(document).off().on('click', '.edit-soal-btn', function() {
+    $('.edit-soal-btn').off().on('click', function() {
```

Comment:
Last time we added a `.off()` to this part, but I didn't notice that this is canceling the `document` event handler. To fix this while ensuring the problem faced last time won't appear again, let jQuery listen only to `.edit-soal-btn` instead of whole `document`. 

Here's the difference:
Old: Select `document` -> call `.off()` -> select `.edit-soal-btn` -> Execute function
New: Select `.edit-soal-btn` -> call `.off()`  -> Execute function